app/models/concerns/search.rb

```diff
@@ -40,6 +26,17 @@ def status(param)
       param == 'active' ? where(active: true) : where(active: false)
     end
 
+    def keyword(k)
+      tsquery = <<-SQL
+        (to_tsquery('english', ''' ' || '#{k}' || ' ''' || ':*'))
```

Comment:
Looks like it eventually delegates here in the case of a `String`: https://github.com/rails/rails/blob/64b09823e6a6b1e19218d3fd815bb65cd2e44f1e/activerecord/lib/active_record/connection_adapters/abstract/quoting.rb#L39